src/main/java/com/simibubi/create/content/trains/entity/TravellingPoint.java

```diff
@@ -88,7 +84,7 @@ public IPortalListener ignorePortals() {
 
 	public ITrackSelector random() {
 		return (graph, pair) -> pair.getSecond()
-			.get(Create.RANDOM.nextInt(pair.getSecond()
+			.get(new Random().nextInt(pair.getSecond()
```

Comment:
this should be reverted.
The proper fix is to get a random from a level or entity, but that needs more refactors.